app/schemas/event_create_schema.rb

```diff
@@ -16,7 +16,7 @@ class EventCreateSchema < JsonSchema
 
     property "zooniverse_user_id" do
       type "string", "integer"
-      pattern "^[0-9]*$"
+      pattern "^.+[0-9]*$"
```

Comment:
i could i just don't want to have to redo. It'll currently fail on finding the user and not just reject the message outright before even attempting to process.